### PR TITLE
bump versions to 7.13.1

### PR DESCRIPTION
**Pull request by rgdoliveira**
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

File: modules/org.kie.kogito.rhpam.builder.prod/module.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: org.kie.kogito.rhpam.builder.prod
-version: "7.13.0"
```
Comment (rgdoliveira) [inline modules/org.kie.kogito.rhpam.builder.prod/module.yaml:2]: I'm not sure about this one, should it be updated?

File: modules/org.kie.kogito.rhpam.bundleinstall/module.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: org.kie.kogito.rhpam.bundleinstall
-version: "7.13.0"
```
Comment (rgdoliveira) [inline modules/org.kie.kogito.rhpam.bundleinstall/module.yaml:2]: I'm not sure about this one, should it be updated?

File: modules/org.kie.kogito.rhpam.builder.prod/module.yaml (hunk 1 above)
Comment (radtriste) [reply modules/org.kie.kogito.rhpam.builder.prod/module.yaml:2]: yes it should IMO

File: modules/org.kie.kogito.rhpam.bundleinstall/module.yaml (hunk 2 above)
Comment (radtriste) [reply modules/org.kie.kogito.rhpam.bundleinstall/module.yaml:2]: yes it should IMO

**Review by radtriste — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Pull request merged**